src/platform/i386/lapic.c

```diff
@@ -320,6 +320,13 @@ lapic_set_page(u32_t page)
 	printk("\tSet LAPIC @ %p\n", lapic);
 }
 
+int
+lapic_spurious_handler(struct pt_regs *regs)
+{
+	printk("\n\n spurious irq happened on cpu%d @ %x\n", get_cpuid(), regs->ip);
```

Comment:
We don't need a printk here.. 

Adding a reference to where I found that spurious vector handlers should not send EOI (last line in the below pasted information) since @yzcode asked in our last meeting:

This is in the Intel® 64 and IA-32 Architectures Software Developer’s Manual => Volume 3 (3A, 3B, 3C & 3D): System Programming Guide =>
(https://software.intel.com/sites/default/files/managed/39/c5/325462-sdm-vol-1-2abcd-3abcd.pdf)
```
10.9 SPURIOUS INTERRUPT
A special situation may occur when a processor raises its task priority to be greater than or equal to the level of the interrupt for which the processor INTR signal is currently being asserted. 
If at the time the INTA cycle is issued, the interrupt that was to be dispensed has become masked (programmed by software), the local APIC will deliver a spurious-interrupt vector. 
Dispensing the spurious-interrupt vector does not affect the ISR, so the handler for this vector should return without an EOI.
```

I've also previously mentioned that the vector number should end with 0xF for Spurious interrupt, but I'm not sure where I read that. It doesn't seem to be in the manual I referenced. Not sure!

Reply:
Thank you for your clarification, I am using 0x3F (63) for the vector number of the Spurious Interrupt now.